backend/main.py

```diff
@@ -26,4 +35,165 @@
     allow_headers=["*"],
 )
 
-app.include_router(api_router, prefix=settings.API_STR)
+
+class Message(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    name: str
+    address: str
+    problem: str
+    number_affected_ppl: str
+    timestamp: Optional[datetime] = Field(
+        sa_column=Column(DateTime(timezone=True), index=True, server_default=func.now())
+    )
+
+
+connect_args = {"check_same_thread": False}
+engine = create_engine("sqlite:///./backend.db", connect_args=connect_args)
+
+
+def init_db():
+    SQLModel.metadata.create_all(engine)
+
+
+@app.on_event("startup")
+def on_startup():
+    init_db()
+
+
+# Messages event generator
+STREAM_DELAY = 5  # In seconds
```

Comment:
@MRigal we could turn this into an env variable, no?

Reply:
I can make it conditional, but there should be a default, as it is not something we're not gonna change frequently. Also adding env variables to the os custom builds adds chances that it gets lost somewhere on the way, it's not a managed service with easy GUI remote config...